### PR TITLE
Add satellite location notebook, rename lonboard folder

### PR DESCRIPTION
**Pull request by wrynearson**
Adding a new notebook (satellite locations, from [here](https://github.com/developmentseed/eoapi-risk/blob/satellite-locations/notebooks/satellite-location.ipynb)).

Renamed `lonboard` folder to address a bug where the library `lonboard` couldn't import.

### CODE REVIEW

**Comment by wrynearson** [comment]
When I try to create a `PathLayer`, it fails with the following error:

`NameError: name 'ArrowDtype' is not defined`

Is this an issue with geodataframes not being supported in JupyterLite?

**Comment by kylebarron** [comment]
> Is this an issue with geodataframes not being supported in JupyterLite?

To pass in a GeoDataFrame you need to import pyarrow. You can get a pyarrow wheel from 
https://github.com/joemarshall/pyarrow-pyodide/releases/tag/0.26.2

**Comment by wrynearson** [comment]
Thank you @kylebarron! This now works 😄

Screen Recording 2024-10-01 at 15 57 22

File: content/lonboard_example/data-filter-extension.ipynb
Comment (kylebarron) [inline content/lonboard_example/data-filter-extension.ipynb:1]: Can you update the link in the README as well?

**Review by kylebarron — APPROVED** [review]

**Pull request merged**